python/QC4_HV.py

```diff
@@ -21,7 +21,7 @@
 cmd.append("--SelectRowEnd= 37")
 cmd.append("--CanvRangeX= 0,1000")
 cmd.append("--CanvRangeY= 0,7")
-cmd.append("--YaxisScale= True")
+cmd.append("--YaxisScale=true")
```

Comment:
This repository is already using optparser in the python tools.  Additionally the python tools for the GEM online software is all based on optparser.  To ensure easier long-term support and uniformity in GEM software please switch to optparser as requested earlier.